client/src/pages/certification.css

```
@@ -145,7 +145,7 @@
 }
 
 .certificate-outer-wrapper {
-  margin-top: calc(-1 * var(--header-height));
+  margin-top: -1 * var(--header-height);
```

You need to remove the math associated with `calc` as well
```suggestion
  margin-top: var(--header-height);
```